src/glmfit.jl

```diff
@@ -603,8 +614,9 @@ $FIT_GLM_DOC
 """
 glm(X, y, args...; kwargs...) = fit(GeneralizedLinearModel, X, y, args...; kwargs...)
 
-GLM.Link(r::GlmResp) = r.link
-GLM.Link(m::GeneralizedLinearModel) = Link(m.rr)
+Link(r::GlmResp) = r.link
+Link(m::GeneralizedLinearModel) = Link(m.rr)
+Link(m::StatsModels.TableRegressionModel{<:GeneralizedLinearModel}) = Link(m.model)
```

Comment:
I'm a bit surprised that you need to add this, as usually method delegations were only done in StatsModels, for functions from StatsAPI. For example, below `Distribution` isn't defined for `TableRegressionModel`. Where is this method called from?

Reply:
`Link` is owned by GLM.

Reply:
They're called in the tests.

Reply:
Yeah but I wonder why we never needed this kind of definition before. I think the code was designed so that we never called functions on objects which could either be `AbstractGLM`s or `TableRegressionModel`s.

Maybe this doesn't matter much as 1.x is a dead-end anyway.